packages/yourloops/test/components/password/password-confirm.test.tsx

```diff
@@ -0,0 +1,137 @@
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import { act } from "@testing-library/react-hooks/dom";
+import { ThemeProvider } from "@material-ui/styles";
+
+import { mainTheme } from "../../../components/theme";
+import { PasswordConfirm } from "../../../components/password/password-confirm";
+import PasswordLeakService from "../../../services/password-leak";
+
+function TestPasswordConfirm(): void {
+
+  describe("Confirm password", () => {
+
+    let container: HTMLElement | null = null;
+    const onErrorStub = sinon.stub();
+    const onSuccessStub = sinon.stub();
+    let passwordLeakService: sinon.SinonStub;
+    const securedPassword = "ThisPasswordIsSecured:)";
+
+    const mountComponent = async (): Promise<void> => {
+      await act(() => {
+        return new Promise((resolve) => {
+          render(
+            <ThemeProvider theme={mainTheme}>
+              <PasswordConfirm
+                onError={() => onErrorStub()}
+                onSucces={(passwordToUse) => onSuccessStub(passwordToUse)}
+              />
+            </ThemeProvider>, container, resolve);
+        });
+      });
+    };
+
+    beforeEach(() => {
+      container = document.createElement("div");
+      document.body.appendChild(container);
+    });
+
+    afterEach(() => {
+      if (container) {
+        unmountComponentAtNode(container);
+        container.remove();
+        container = null;
+      }
+    });
+
+    function setPasswords(password: string, confirmPassword: string) {
+      const event = new Event("input", { bubbles: true });
+      const nativeInputValueSetter = Object.getOwnPropertyDescriptor(window.HTMLInputElement.prototype, "value").set;
+      const passwordInput = document.getElementById("password") as HTMLInputElement;
+      nativeInputValueSetter.call(passwordInput, password);
+      passwordInput.dispatchEvent(event);
+
+      const confirmPasswordInput = document.getElementById("confirm-password") as HTMLInputElement;
+      nativeInputValueSetter.call(confirmPasswordInput, confirmPassword);
+      confirmPasswordInput.dispatchEvent(event);
```

Comment:
Could be simplified with "Simulate" from react test utils 
```suggestion
      const passwordInput = document.getElementById("password") as HTMLInputElement;
      const confirmPasswordInput = document.getElementById("confirm-password") as HTMLInputElement;
      
      Simulate.change(passwordInput, { target: { value: password } } as unknown as SyntheticEventData);
      Simulate.change(confirmPasswordInput, { target: { value: confirmPassword } } as unknown as SyntheticEventData);
```